luautils/lua.go

```diff
@@ -6,12 +6,14 @@ import (
 
 // Processor handles Lua script processing
 type Processor struct {
-	L        *lua.LState
 	BasePath string
 }
 
-// NewProcessor creates a new Lua script processor
 func NewProcessor(path string) *Processor {
+	return &Processor{BasePath: path}
```

Comment:
fwiw you definitely couldn't have multiple threads share one instance of this object still